odoo/cli/tsconfig.py

```diff
@@ -39,93 +41,89 @@ def run(self, cmdargs):
             description=self.__doc__
         )
 
+        parser.add_argument('--addons-path', type=str, nargs='+', dest="paths")
+
         args, unknown = parser.parse_known_args(args=cmdargs)
+        paths = args.paths
 
         modules = {}
-        for path in unknown:   
-            for module in self.get_module_list(self.clean_path(f"{path}")):
-                modules[module] = self.prefix_suffix_path(module, f"{path}", "/static/src/*")
+        for path in paths:
+            for module in self.get_module_list(self.clean_path(path)):
+                modules[module] = self.prefix_suffix_path(module, path, "/static/src/*")
 
         content = self.generate_file_content(modules)
-
-        print(content)
+        print(json.dumps(content, indent=2))
 
     def generate_imports(self, modules):
-        return [f""""@{module}/*": ["{path}"]""" for _, (module, path) in enumerate(modules.items())]
-
-    def generate_includes(self, modules):
-        return [f"""\"{path.replace("src/*", "**/*")}.js\"""" for _, (module, path) in enumerate(modules.items())]
+        return dict([("@%s/*" % module, ["%s" % path]) for module, path in modules.items()])
```

Comment:
Python has more than just list comprehensions. Here there's no reason to create a list to convert to a dict, you can create the dict directly:
```python
return {
    f'@{module}/*': [path] # not sure why you're converting it back to a string? Thoug if that's necessary, use `str`
    for module, path in modules.items()
}
```

Reply:
I had a misconception about the json.dumps. I indeed do not need the additional " "